README.md

```diff
@@ -110,12 +110,42 @@ The latest build of the project can be found [here](https://pypi.org/project/see
 as a wheel file. The file is published as a courtesy and does not imply any guarantee or obligation for support from the
 publisher.
 
+### Pre-requisites
+
+Before installing the **seeq-azureml** Seeq Add-on, you will have to create an **Azure application** and **service
+principal**. Follow the steps
+in [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) to
+create your application, and
+choose [Option 2](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret)
```

Comment:
Is it worth mentioning if option 1 is viable or won't work at all?

Reply:
I think the description currently is clear enough. What I have learned from doing this in the past is that detailing what not to do, can a lot of times lead to more confusion.